mediacapture-streams/MediaStream-MediaElement-preload-none.https.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Test that the HTMLMediaElement preload 'none' attribute value is ignored for MediaStream used as srcObject and MediaStream object URLs used as src.</title>
+        <link rel="author" title="Matthew Wolenetz" href="mailto:wolenetz@chromium.org"/>
```

Comment:
This file seems like it belongs to a different PR.

Reply:
that file was a `-manual` file, presumably because when it was created, one would not gain access to getUserMedia. Now that we do, it feels reasonable to make the move here.